EIPS/eip-common_metrics.md

```diff
@@ -0,0 +1,56 @@
+---
+eip: <to be assigned>
+title: Common Prometheus Metrics Names for Clients
+author: Adrian Sutton (@ajsutton)
+discussions-to: https://ethereum-magicians.org/t/common-chain-metrics/3415/2
+status: Draft
+type: Informational
+created: 2019-07-01
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+Standardized names for common metrics Ethereum clients to use with Prometheus. 
```

Comment:
Can you explain what Prometheus is?

Reply:
Added basic description and link to the Prometheus website for further info.